vignettes/summary_selecting.Rmd

```diff
@@ -0,0 +1,229 @@
+---
+title: "A quick summary of selecting possibilities in `datawizard`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{A quick summary of selecting possibilities in `datawizard`}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup, echo=FALSE, message=FALSE}
+library(datawizard)
+library(poorman)
+library(htmltools)
+
+set.seed(123)
+iris <- iris[sample(nrow(iris), 10), ]
+row.names(iris) <- NULL
+
+row <- function(...) {
+  div(
+    class = "custom_note",
+    ...
+  )
+}
+```
+
+```{css, echo=FALSE}
+.custom_note {
+  border-left: solid 5px hsl(220, 100%, 30%);
+  background-color: hsl(220, 100%, 95%);
+  padding: 5px;
+  margin-bottom: 10px
+}
+```
+
+# Selecting variables
+
+## Quoted names
+
+This is the most simple way to select one or several variables. Just use a character
+vector containing variables names, like in base R.
+
+```{r}
+get_columns(iris, c("Sepal.Length", "Petal.Width"))
+```
+
+## Unquoted names
+
+It is also possible to use unquoted names. This is useful if we use the `tidyverse`
+and want to be consistent about the way variable names are passed.
+
+```{r}
+iris %>%
+  group_by(Species) %>%
+  standardise(Petal.Length) %>%
+  ungroup()
+```
+
+
+## Formulas
+
+You can also use formulas to select variables:
+
+```{r}
+get_columns(iris, ~ Sepal.Length + Petal.Width)
+```
+
+This was mostly useful to use selection more easily in custom functions before
+`datawizard` 0.6.0, and is kept available for backward compatibility. 
```

Comment:
Do we think this is the best practice? If not, we don't need to mention this possibility.

Reply:
This could be moved to the "useful to know" part. Since this use is not really deprecated (no message, nothing in the news), it would be weird to erase it completely from the docs.

Reply:
Yes, that sounds better to me.